decidim-core/app/controllers/decidim/devise/passwords_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
```

Comment:
Needed to ease the transition from Ruby 2.3.1 to 3. Strings are immutable by default in Ruby 3.